### PR TITLE
Add all declarations to exports

### PR DESCRIPTION
**Pull request by Lucianovici**
Why not exporting everything?
I'd like to override some templates and use your components in a different way.

Is there any risk?

### CODE REVIEW

**Comment by mattlewis92** [comment]
My reason for not exporting the child components right now is because I'm not 100% sure of their API as they will probably be refactored slightly when https://github.com/mattlewis92/angular-calendar/issues/16 lands, which will introduce some breaking changes. 

My plan for allowing custom templates in an AoT friendly way is to make use of the `ngComponentOutlet` directive so that you'd be able to do
```
import { Component, Type } from '@angular/core';
import { CalendarMonthCellComponent } from 'angular-calendar';

@Component({
  template: 'custom month view template'
})
class MyMonthCellComponent extends CalendarMonthCellComponent {}

@Component({
  <mwl-calendar-month-view [monthCellComponent]="monthCellComponent"></mwl-calendar-month-view>
})
class MyComponentThatUsesTheCalendar {
  monthCellComponent: Type<any> = MyMonthCellComponent;
}
```

If you're happy to accept that there will probably be a lot of breaking changes WRT to the child components in the next release then I'm happy to merge this now though.

File: src/calendar.module.ts [hunk 1]
```diff
@@ -66,13 +66,25 @@ import { CalendarDateFormatter } from './providers/calendarDateFormatter.provide
...
   ],
   exports: [
-    CalendarDayViewComponent,
-    CalendarWeekViewComponent,
-    CalendarMonthViewComponent,
-    CalendarDatePipe,
-    CalendarPreviousViewDirective,
-    CalendarNextViewDirective,
-    CalendarTodayDirective
+      CalendarDayViewComponent,
```
Comment (mattlewis92) [inline src/calendar.module.ts]: Please fix the whitespace to match the rest of the file

**Comment by mattlewis92** [comment]
Can you also change the commit message to be `feat: add all declarations to exports`, otherwise it won't show up in the changelog.

**Comment by Lucianovici** [comment]
Alright. Thanks! I created another PR, (since I reverted the last commit).

**Pull request closed without merging**